.github/workflows/lint.yml

```diff
@@ -20,9 +20,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           show-progress: false
-      - name: Configure git hooks
-        # Configure git to quell an irrelevant warning for runners (they never commit / push).
-        run: git config core.hooksPath githooks
```

Comment:
Nit: This job does not need to install the LLVM toolchain, so adding the shared config will only slow it down, can we leave the file as-is?